src/components/Hero.tsx

```diff
@@ -4,20 +4,24 @@ import classNames from 'classnames'
 
 import { Background } from '@components'
 
-interface ProductHero extends Background {
+interface Hero extends Background {
     product?: 'code search' | 'batch changes' | 'code insights'
     title: string | ReactNode
     backButton?: ReactNode
+    leftCol?: ReactNode
+    subtitle?: string
     description?: string
     cta?: ReactNode
     displayUnderNav?: boolean
 }
 
-export const ProductHero: FunctionComponent<Omit<ProductHero, 'className' | 'children' | 'illustration'>> = ({
+export const Hero: FunctionComponent<Omit<Hero, 'className' | 'children' | 'illustration'>> = ({
     variant,
     product,
     title,
     backButton,
+    leftCol,
```

Comment:
Just a thought, but we may be able to make use of our TwoColumnSection

Reply:
Good idea, ok I shifted things around to use that component in [e0580df](https://github.com/sourcegraph/about/pull/5658/commits/e0580df2fc4a5d77f3423419c49a273af237cbf1). Let me know what you think!